command/jobs/base.go

```diff
@@ -106,8 +106,6 @@ func (f *FirehoseBase) watch(w WatchJobListFunc) {
 		AllowStale: true,
 	}
 
-	newMax := f.lastChangeIndex
```

Comment:
`newMax` is always referencing `f.lastChangeIndex`

`f.lastChangeIndex` is only changed in this function, outside the job range, so no need to copy the value into another property